src/test/java/org/folio/des/scheduling/acquisition/AcqBaseExportTaskTriggerTest.java

```diff
@@ -451,7 +452,7 @@ void hourlyScheduleWithHours(int frequency, int addHours, int expDiffHours) {
     Instant actInstant = Instant.ofEpochMilli(actDate.getTime());
     ZonedDateTime actZonedDateTime = ZonedDateTime.ofInstant(actInstant, ZoneId.of("UTC"));
     int actHour = actZonedDateTime.getHour();
-    assertEquals(actHour, nowHour + expDiffHours + addHours);
+    assertEquals((nowHour + expDiffHours + addHours) % 24, actHour);
   }
```

Comment:
in fail case, expected was 20 + 2 + 4 = 26 which is not correct hour, so it changed to in 24 hours format (20 + 2 + 4) % 24 = 2